nilearn/datasets/func.py

```diff
@@ -998,11 +997,11 @@ def _is_valid_path(path, index, verbose):
 
     # Load covariates file
     participants_file = os.path.join(data_dir, participants_file)
-    csv_data = pd.read_csv(participants_file, delimiter="\t")
-    csv_data = csv_data[csv_data["participant_id"].isin(subject_ids)]
+    csv_data = np.recfromcsv(participants_file, delimiter="\t")
```

Comment:
PEP8 recommends using single-quotes for strings everywhere, except when one has to enclose single quotes.

Reply:
On the other hand I have noticed recently that [_Black_ prefers double quotes over single quotes](https://black.readthedocs.io/en/stable/the_black_code_style.html#strings).